doc/source/release.1.9.1.rst

```diff
@@ -0,0 +1 @@
+.. include:: ../release/1.9.1-notes.rst
```

Comment:
Can I propose to make the change to not have a need for this extra file?

Reply:
at least not in this branch. in `main` yes you can propose it. I personally get some use out of all the release notes being in a separate directory, but maybe that can be done without needing these one-line files.

Reply:
I would just like to clean `doc/source`. NumPy has a single place for release files in `doc/source/release` for instance.